actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -250,11 +248,10 @@ pub fn call_generic<RT: Runtime>(
                         };
                         // TODO: support IPLD codecs #758
                         let params = RawBytes::serialize(BytesSer(input_data))?;
-                        if kind == CallKind::StaticCall {
-                            system.send_read_only(&dst_addr, method, params)
-                        } else {
-                            system.send(&dst_addr, method, params, TokenAmount::from(&value))
-                        }
+                        let value = TokenAmount::from(&value);
+                        let gas_limit = if !gas.is_zero() { Some(gas.as_u64()) } else { None };
```

Comment:
This panics if gas is too big. We probably want to make this "none" if gas > i64::max or 0.

Reply:
god damn it, I'll make a from method for u64 that just truncates the thing, I've had enough with these hidden panics.

Reply:
added `U256::to_u64_saturating` to fix this once and for all future uses.